Source/WebCore/style/StyleAdjuster.cpp

```diff
@@ -610,6 +610,9 @@ void Adjuster::adjust(RenderStyle& style, const RenderStyle* userAgentAppearance
     style.adjustAnimations();
     style.adjustTransitions();
 
+    // Do the same for scroll-timeline longhands.
```

Comment:
// FIXME ?

Reply:
Not a fixme, this comment makes sense in the context of code further up.